tools/Generator.CLI/Program.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+namespace Generator.CLI
+{
+	class Program
+	{
+		static int Main(string[] args)
+		{
+			var runner = new CommandDotNet.AppRunner<ConsoleShell>();
+#if (DEBUG)
+			runner.Run(args);
+			while (true)
+			{
+				var input = Console.ReadLine();
+				if (string.IsNullOrEmpty(input))
+					return 0;
+
+				args = input.Split(' ');
+				Console.Clear();
+				var state = runner.Run(args);
+				Console.WriteLine($"Application finished with {state}.");
+			}
+#else
+			return runner.Run(args);
+#endif
+
+		}
```

Comment:
.editorsettings can warn for these blank lines? That's cool! Can you point me to the setting?

Reply:
I would imagine it can streamline the formatting behavior for them. I'll try look it up. Possibly could also configure roslynquoter to fix such things. I think i've read that there are some MS teams enforcing coding styles that way.